Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -3723,17 +3723,17 @@ - (ORKOrderedTask *)makeWaitingTask {
     return waitTask;
 }
 
-- (void)updateProgress:(CGFloat)progress OfWaitTask:(ORKWaitStepViewController *)viewController {
+- (void)updateProgress:(CGFloat)progress waitStepViewController:(ORKWaitStepViewController *)viewController {
     if (progress <= 1.0) {
         [viewController setProgress:progress animated:true];
         
         double delayInSeconds = 0.1;
         dispatch_time_t popTime = dispatch_time(DISPATCH_TIME_NOW, (int64_t)(delayInSeconds * NSEC_PER_SEC));
-        __weak ORKWaitStepViewController *vc = viewController;
+        __weak ORKWaitStepViewController *weakViewController = viewController;
```

Comment:
Done. I also removed the `__weak` conversion for `self` (since the block is temporary and not permanently retained locally) and improved the float comparison.